workflows/manage-service-now-ticket/ManageServiceNowTicketwithWorkflows.json

```diff
@@ -0,0 +1,233 @@
+{
+	"name": "Manage Service Now Ticket with Workflows",
+	"description": "Manage Service Now Ticket with Workflows",
+	"definition": {
+		"start": "Get Identity",
+		"steps": {
+			"Compare Numbers": {
+				"choiceList": [
+					{
+						"comparator": "NumericEquals",
+						"nextStep": "Send Email",
+						"variableA.$": "$.manageServiceNowTicket.statusCode",
+						"variableB": 201
+					}
+				],
+				"defaultStep": "Send Email 3",
+				"description": "Check Manage Service Now Ticket action completed successfully",
+				"type": "choice"
+			},
+			"Compare Strings": {
+				"choiceList": [
+					{
+						"comparator": "StringEquals",
+						"nextStep": "Send Email 1",
+						"variableA.$": "$.manageServiceNowTicket1.body.result[?(@.task_effective_number=='{{$.manageServiceNowTicket.body.result.task_effective_number}}')].request_state",
+						"variableB": "closed_complete"
+					}
+				],
+				"defaultStep": "Send Email 2",
+				"description": null,
+				"type": "choice"
+			},
+			"Define Variable": {
+				"attributes": {
+					"variables": [
+						{
+							"name": "ticketDueDate",
+							"transforms": [
+								{
+									"id": "sp:transform:addTime:time",
+									"input": {
+										"length": 7,
+										"type": [
+											"string"
+										],
+										"unit": "days"
+									}
+								}
+							],
+							"variableA.$": "$.now()"
+						}
+					]
+				},
+				"description": "Due Date variable = 7 days from today",
+				"nextStep": "Define Variable 1",
+				"type": "Mutation"
+			},
+			"Define Variable 1": {
+				"attributes": {
+					"variables": [
+						{
+							"name": "dueDate",
+							"transforms": [
+								{
+									"id": "sp:transform:substring:string",
+									"input": {
+										"length": 10,
+										"start": null,
+										"type": [
+											"string"
+										]
+									}
+								}
+							],
+							"variableA.$": "$.defineVariable.ticketDueDate"
+						}
+					]
+				},
+				"description": "Due Date variable = 7 days from today",
+				"nextStep": "Manage ServiceNow Ticket",
+				"type": "Mutation"
+			},
+			"End Step — Success": {
+				"description": "Request Success",
+				"type": "success"
+			},
+			"End Step — Success 1": {
+				"type": "success"
+			},
+			"End Step — Success 2": {
+				"type": "success"
+			},
+			"Get Identity": {
+				"actionId": "sp:get-identity",
+				"attributes": {
+					"id.$": "$.trigger.attributes.manager.id"
+				},
+				"description": "Get User's Manager",
+				"nextStep": "Define Variable",
+				"type": "action",
+				"versionNumber": 2
+			},
+			"Manage ServiceNow Ticket": {
+				"actionId": "sp:snow",
+				"attributes": {
+					"action": "create",
+					"authenticationType": "basic",
+					"basicAuthPassword": "$.secrets.9579f656-d11a-422d-9c0a-df704252737e",
```

Comment:
Please remove any reference to `$.secrets` from your workflows. Having these here will cause users to receive an error message when they try to import the workflow.  You just need empty quotes.

Reply:
Removed the $.secrets references from this workflow